CHANGELOG.md

```diff
@@ -18,7 +18,7 @@
 
 ### `Deprecated`
 
-## v1.5 nf-core/mhcquant "Flying Fish" - 2020/04/18
+## v1.5.0 nf-core/mhcquant "Flying Fish" - 2020/04/18
```

Comment:
```suggestion
## v1.5 nf-core/mhcquant "Flying Fish" - 2020/04/18
```
However the old ones should probably stay like this as otherwise people would not be able to access them, sorry!